app/jobs/lessons/import_all_content_job.rb

```diff
@@ -0,0 +1,10 @@
+module Lessons
+  class ImportAllContentJob
+    include Sidekiq::Worker
+    sidekiq_options retry: 1, dead: false
+
+    def perform
+      LessonContentImporter.import_all
+    end
+  end
+end
```

Comment:
No spec for this one since there's nothing to test that's not covered by Sidekiq or `LessonContentImporter`